src/acquisition/ght/ght_update.py

```diff
@@ -334,6 +340,7 @@ def main():
   parser.add_argument('term', action='store', type=str, default=None, help='term/query/topic (ex: all; /m/0cycc; "flu fever")')
   parser.add_argument('--first', '-f', default=None, type=int, help='first epiweek override')
   parser.add_argument('--last', '-l', default=None, type=int, help='last epiweek override')
+  parser.add_argument('--country', '-c', default='US', type=string, help='location country (ex: US; BR)')
```

Comment:
This should be `str` instead of `string`

Reply:
Done, thanks!